boms/cloud-lts-bom/pom.xml

```diff
@@ -7,7 +7,7 @@
 
   <groupId>com.google.cloud</groupId>
   <artifactId>gcp-lts-bom</artifactId>
-  <version>1.0.11-SNAPSHOT</version>
+  <version>1.0.12-SNAPSHOT</version>
```

Comment:
Is this just updating snapshot version post-release?
The title somehow doesn't make sense to me when I look at the actual change.